Detector/DetFCCeeIDEA/compact/DriftChamber.xml

```diff
@@ -0,0 +1,36 @@
+<lccdd>
+<!--
+    (c) Copyright 2021 CERN for the benefit of the LHCb Collaboration
```

Comment:
```suggestion
    (c) Copyright 2021 CERN for the benefit of the LHCb Collaboration
```
I don't think LHCb needs to hold the copyright here. we don't have the copyright notice in the other xmls, for licensing it would be just good if you agree that your contributions will be licenced under Apache 2.0 in the future.

Reply:
@vvolkl copyright removed